src/utils/tests/json.rs

```diff
@@ -0,0 +1,88 @@
+use crate::utils::json::default_if_null;
+use serde::Deserialize;
+
+
+#[derive(PartialEq, Deserialize, Debug)]
+struct ExpectedStruct {
+    #[serde(deserialize_with = "default_if_null")]
+    field: String,
```

Comment:
Maybe u64 would have been more interesting for us, but either way this is nice :)